python/ray/worker.py

```diff
@@ -2097,7 +2097,12 @@ def kill(actor: "ray.actor.ActorHandle", *, no_restart: bool = True):
 
 @PublicAPI
 @client_mode_hook(auto_init=True)
-def cancel(object_ref: ray.ObjectRef, *, force: bool = False, recursive: bool = True):
+def cancel(
+    object_refs: Union[ray.ObjectRef, List[ray.ObjectRef]],
+    *,
+    force: bool = False,
+    recursive: bool = True,
+):
     """Cancels a task according to the following conditions.
```

Comment:
thanks for cleaning up the apis! As @rkooo567 mentioned, we need to clear on the semantics on what to expect it returns, especially around partial failures.

And we need good test coverage for those semantics.